src/Wallabag/CoreBundle/Resources/views/Entry/share.html.twig

```diff
@@ -17,12 +17,8 @@
         <meta name="twitter:site" content="@wallabagapp" />
         <meta name="twitter:title" content="{{ entry.title|e|raw }}" />
         <meta name="twitter:description" content="{{ entry.content|striptags|slice(0, 300)|raw }}&hellip;" />
-        {% if app.debug %}
-            <script src="{{ asset('wallassets/public.dev.js') }}"></script>
-        {% else %}
-            <link rel="stylesheet" href="{{ asset('wallassets/public.css') }}">
-        {% endif %}
 
+        {{ encore_entry_link_tags('public', null, app.environment == 'dev' ? 'dev' : 'prod') }}
```

Comment:
now test environment is using prod assets that are generated.
did this change because on CI dev assets are not generated, and it can happen that on dev we don't have them neither, so using prod assets ensure we don't have assets missing errors when running tests